### PR TITLE
Add steps for CVE ID and CVE details

### PR DESCRIPTION
**Pull request by ritazh**

### CODE REVIEW

File: security-release-process.md [hunk 1]
```diff
@@ -161,7 +161,7 @@ Note: The kubernetes-security GitHub org is co-owned and viewable by the SRC and
 These steps should be completed within the 1-7 days of Disclosure.
 
 - The Fix Lead and the Fix Team will create a [CVSS](https://www.first.org/cvss/specification-document) score using the [CVSS Calculator](https://www.first.org/cvss/calculator/3.0). They will also use the [Severity Thresholds - How We Do Vulnerability Scoring](#severity-thresholds---how-we-do-vulnerability-scoring) to determine the effect and severity of the bug. The Fix Lead makes the final call on the calculated risk; it is better to move quickly than make the perfect assessment.
-- The Fix Lead will request a CVE from the [Kubernetes CVE Numbering Authority](cve-requests.md).
+- The Fix Lead will [Assign a CVE ID to vulnerability from the CVE Numbering Authority](/cna-handbook.md#assign-a-cve-id-to-vulnerability).
```
Comment (cji) [inline security-release-process.md:164]:
```suggestion
- The Fix Lead will [Assign a CVE ID to the vulnerability from the CVE Numbering Authority](/cna-handbook.md#assign-a-cve-id-to-vulnerability).
```
just a nit, think we need to update the title of the linked section too. But we can do that in another PR if you'd like.

**Review by cji — APPROVED** [review]
LGTM, typo nit but not a blocker

File: security-release-process.md (hunk 1 above)
Comment (ritazh) [reply security-release-process.md]: done. PTAL

Comment (cji) [reply security-release-process.md]: LGTM! Thanks!

File: cna-handbook.md [hunk 2]
```diff
@@ -96,64 +99,66 @@ To see help output on creating a new CNA admin user, you can run the following c
...
-Once the upstream pull request is merged, indicate in the tracking sheet that the CVE has been
-populated.
+1. Once a vulnerability is made public, populate the CVE details in https://github.com/CVEProject/cvelist. This should be done as soon as reasonably possible after the vulnerability is made public (ideally, O(days)).
+1. Generate the CVElist json file
+
+    * See https://github.com/CVEProject/cvelist/blob/master/2023/2xxx/CVE-2023-2727.json as an example.
+
+    * https://vulnogram.github.io/#editor is a useful tool for generating the CVE details, but should only be used once the vulnerability has been made public.
+
+    Fill in or update the relevant fields, including:
```
Comment (SaranBalaji90) [inline cna-handbook.md:130]: Are all these fields accurate? I don't see some of these in vulnogram editor.

File: cna-handbook.md [hunk 3]
```diff
@@ -96,64 +99,66 @@ To see help output on creating a new CNA admin user, you can run the following c
...
-SRC member.
-
-Once the PR to our fork has merged, a CNA approved member should open a PR to sync the upstream
-cvelist with our fork:
-https://github.com/CVEProject/cvelist/compare/master...kubernetes-security:master
-
-
-Once the upstream pull request is merged, indicate in the tracking sheet that the CVE has been
-populated.
+1. Once a vulnerability is made public, populate the CVE details in https://github.com/CVEProject/cvelist. This should be done as soon as reasonably possible after the vulnerability is made public (ideally, O(days)).
```
Comment (SaranBalaji90) [inline cna-handbook.md:123]: I'm guessing CVE details gets auto-populated now?

File: cna-handbook.md (hunk 2 above)
Comment (ritazh) [reply cna-handbook.md:130]: I took these fields from the 4.0 json generated and they should all be in vulnogram editor. Will update `workaround` so it has a space like it is in the editor. The only field that is in 4.0 but not in 5.0 is `Assigner`. But I don't know if we can move to 5.0 yet?

File: cna-handbook.md (hunk 3 above)
Comment (ritazh) [reply cna-handbook.md:123]: AFAICT after we run `cvelib publish` the information gets auto-populated on https://github.com/CVEProject/cvelist

File: security-release-process.md [hunk 4]
```diff
@@ -225,13 +225,14 @@ Communications process:
   - [discuss.kubernetes.io](https://discuss.kubernetes.io/c/announcements) forum (this should be
     posted automatically using the general announcement email template)
   - Tracking issue opened in https://github.com/kubernetes/kubernetes/issues
-    ([template](comms-templates/vulnerability-announcement-issue.md)) and prefixed with the
-    associated CVE ID (if applicable)
+    ([template](comms-templates/vulnerability-announcement-issue.md)) and prefixed with the 
+    associated CVE ID (if applicable). Add `/label official-cve-feed` so it will be part of https://kubernetes.io/docs/reference/issues-security/official-cve-feed/. Close the issue after the announcement is made.
```
Comment (ritazh) [inline security-release-process.md:229]: @SaranBalaji90 Added this step based on yesterday's feedback meeting

File: cna-handbook.md [hunk 5]
```diff
@@ -96,64 +99,66 @@ To see help output on creating a new CNA admin user, you can run the following c
...
+
+    Fill in or update the relevant fields, including:
+    * State: `PUBLIC`
+    * Assigner: `security@kubernetes.io`
+    * Affects: List historical minor versions affected and "prior to 1.x.y" patch versions affected
+    * Credits: The vulnerability reporter's name
+    * Description: `The Kubernetes <component> command/component in versions <affected versions> <vulnerability>`
+    * Impact: CVSS impact
+    * Problem type: Select a problem type from https://cwe.mitre.org/data/definitions/699.html if applicable
+    * References: Link to the Kubernetes github issue(s) (with type CONFIRM) and mailing list announcement (with type MLIST)
```
Comment (cji) [inline cna-handbook.md:138]:
```suggestion
    * References: Link to the Kubernetes github issue(s) (with type `issue-tracking`) and mailing list announcement (with type `mailing-list`)
```
running through this now and the tags in the tool may have changed.

**Comment by cji** [comment]
Ran through these instructions to publish https://www.cve.org/CVERecord?id=CVE-2023-5528 - thank you!

File: cna-handbook.md (hunk 5 above)
Comment (ritazh) [reply cna-handbook.md]: updated

**Pull request merged**